src/main/java/org/overture/ego/controller/PolicyController.java

```diff
@@ -28,7 +34,9 @@
 public class PolicyController {
 
   @Autowired
-  private AclEntityService aclEntityService;
+  private PolicyService policyService;
+  private GroupService groupService;
+  private UserService userService;
```

Comment:
You are missing autowired on the other services.

Lets use an autowired constructor instead of field injection. This will also allow us to declare our dependencies as final. 

```java
  private final PolicyService policyService;
  private final GroupService groupService;
  private final UserService userService;

  @Autowired
  public PolicyController(PolicyService policyService, GroupService groupService, UserService userService) {
    this.policyService = policyService;
    this.groupService = groupService;
    this.userService = userService;
  }
```